### ISSUE TITLE
Expert trajactory performance

### ISSUE THREAD

**Issue by 1215481871** [GeorgeCazenavette/mtt-distillation#8]
Thanks for your work! I've got a question. When training the expert trajactory with CiFAR10 accroding to buffer.py, I only got test accuracy around 0.79 and 0.77 w/o --zca after 50 epochs. However, Table 1 in your paper reports that full dataset can reach 0.84 accuracy on CiFAR10. Is there any mistake I've made here?

**Comment by GeorgeCazenavette** [comment]
No mistake :)

Those full dataset numbers were obtained using Adam with a learning rate schedule and weight decay. (These values were reported by the DC/DSA/DM authors.)

Our expert trajectories were trained  using SGD (no momentum) with a constant learning rate and no weight decay since this was much easier for our synthetic data to imitate in our optimization loop.

Does that make sense?

**Comment by 1215481871** [comment]
Thank for your kind reply!

**Comment by 42Shawn** [comment]
That's very helpful!
